sharpy/postproc/aeroforcescalculator.py

```diff
@@ -96,7 +92,7 @@ def run(self, online=False):
             # create folder for containing files if necessary
             if not os.path.exists(self.folder):
                 os.makedirs(self.folder)
-            self.folder += self.settings['text_file_name']
+            self.folder += 'aeroforces.txt'
```

Comment:
This change looks like a loss of functionalities, is there a reason for it?

Reply:
I can't think of a good reason, so it's going back